lib/home/profile/report/issue_report.dart

```diff
@@ -34,6 +25,13 @@ class _IssueReportScreenState extends State<IssueReportScreen> {
     final style = MaterialStateTextStyle.resolveWith((states) => StyleText(
           color: colors.onBackground.withOpacity(states.isFocused ? 1.0 : 0.5),
         ));
+    final border = MaterialStateOutlineInputBorder.resolveWith(
+      (states) => OutlineInputBorder(
+        borderSide: states.isFocused
+            ? BorderSide(color: colors.primary, width: 2)
+            : BorderSide(color: colors.onBackground.withOpacity(0.5)),
+      ),
+    );
```

Comment:
Since this was used in 3 spots, I thought it might be nice to put it in a variable.

Also, no worries if these `MaterialStateProperty` classes seem a little convoluted—I (and many others) can relate to that feeling :)